Source/Scene/UrlTemplateImageryProvider.js

```diff
@@ -132,6 +132,13 @@ define([
      *        source does not support picking features or if you don't want this provider's features to be pickable. Note
      *        that this can be dynamically overridden by modifying the {@link UriTemplateImageryProvider#enablePickFeatures}
      *        property.
+     * @param {Object} [options.customTags] Allow to replace custom keywords in the URL template. The object must have strings as keys and functions as values.
+              For example:
```

Comment:
See the [Documentation guide for examples](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/DocumentationGuide#examples) so this is formatted nicely in the reference documentation.